package.json

```diff
@@ -52,7 +52,10 @@
     "betterer": "betterer",
     "betterer:merge": "betterer merge",
     "betterer:stats": "ts-node --transpile-only --project ./scripts/cli/tsconfig.json ./scripts/cli/reportBettererStats.ts",
-    "betterer:issues": "ts-node --transpile-only --project ./scripts/cli/tsconfig.json ./scripts/cli/generateBettererIssues.ts"
+    "betterer:issues": "ts-node --transpile-only --project ./scripts/cli/tsconfig.json ./scripts/cli/generateBettererIssues.ts",
+    "cy:run": "cypress run --browser=chrome",
```

Comment:
What is the difference between these and the several `e2e:*` commands we have? can we unify them somehow to avoid having many commands doing similar things?

Reply:
these are the commands that exist in the `@grafana/e2e` package currently. they're run by the `run-suite` script in core. they need to be yarn scripts else cypress can't find the installed modules (assume that's a yarn pnp thing). 

re: refactoring - yes, i absolutely think we should refactor these. i was hoping to do it in a follow up PR since i think it would also require changes to drone

Reply:
@ashharrison90 that makes sense. Could you please open an issue to keep track of it? Just so we don't forget. Sadly we can't write comments in this package.json

Reply:
made this issue to capture some followup items: https://github.com/grafana/grafana/issues/74459